README.md

```diff
@@ -42,16 +96,24 @@ For example, a row in the data file could look like this:
 
 ``Optimal investment policy of the regulated firm\tConcept0:0.5,Concept1:1\tConcept0,Concept3``
 
-To train a model, use the ``main`` module inside ``qualle``, e.g.:
+Alternatively, you can specify an [Annif](https://github.com/NatLibFi/annif) (tested with version 0.58)
+full-text document corpus combined with the result of the Annif index method applied on the corpus.
+This is a  folder with three files per document:
 
-``python -m qualle.main train /path/to/train_data_file /path/to/output/model``
+* ``doc.annif`` : result of Annif index method
+* ``doc.tsv`` : ground truth
+* ``doc.txt`` : document content
+
+### Train
+To train a model, use the ``train`` mode, e.g.:
+
+``qualle train /path/to/train_data_file /path/to/output/model``
 
 It is also possible to use label calibration using the subthesauri of a thesaurus (such as the [STW](http://zbw.eu/stw/version/latest/about))
 as categories (please read the paper for more explanations). Consult the help (see above) for the required options.
```

Comment:
vielleicht trotzdem "label calibration (comparison of predicted vs actual labels)"